docs/source/top_level_elements/relatedItem.rst

```diff
@@ -0,0 +1,598 @@
+      </titleInfo>
+    </relatedItem>
+
+The relationship expressed here is structural in nature and should be reflected by default behavior in our DAMS.
+
+relatedItem[@type = "host"][@displayLabel = "Project"]
+------------------------------------------------------
+
+This XPath is typically used to indicate the digital project/digital collection for a given object; e.g. `roth:3346 <https://digital.lib.utk.edu/collections/islandora/object/roth:3346/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <relatedItem displayLabel="Project" type="host">
+      <titleInfo>
+        <title>Albert "Dutch" Roth Photograph Collection</title>
+      </titleInfo>
+    </relatedItem>
+
+The relationship expressed here is structural in nature and should be reflected by default behavior in our DAMS.
+
+relatedItem[@type = "host"][@displayLabel = "Collection"]
+---------------------------------------------------------
+
+This XPath is typically used to indicate the archival collection for a given object; e.g. `heilman:261 <https://digital.lib.utk.edu/collections/islandora/object/heilman:261/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <relatedItem type="host" displayLabel="Project">
+      <titleInfo>
+        <title>Botanical Photography of Alan S. Heilman</title>
+      </titleInfo>
+    </relatedItem>
+    <relatedItem type="host" displayLabel="Collection">
+      <titleInfo>
+        <title>Botany Department Photographs</title>
+      </titleInfo>
+      <identifier type="local">AR.0488</identifier>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix dbo: <http://dbpedia.org/ontology/> .
+
+    <https://example.org/objects/1> dbo:collection "Botany Department Photographs, AR.0488" .
+
+relatedItem[@type = "host"][@displayLabel = "project"]
+------------------------------------------------------
+
+This XPath is used 798 times and only appears in the Thompson Brothers Photograph Collection; e.g. `thompson:1 <https://digital.lib.utk.edu/collections/islandora/object/thompson:1/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <relatedItem type="host" displayLabel="project">
+      <titleInfo>
+        <title>Thompson Brothers Commercial Photographers</title>
+      </titleInfo>
+    </relatedItem>
+
+The relationship expressed here is structural in nature and should be reflected by default behavior in our DAMS.
+
+relatedItem[@type = "host"][@displayLabel = "Digital Collection"]
+-----------------------------------------------------------------
+
+This XPath is used 362 times in the Children's Defense Fund collection: e.g. `cdf:7850 <https://digital.lib.utk.edu/collections/islandora/object/cdf:7850/datastream/MODS/view>`_. Synonymous with :code:`@displayLabel = "Project"`.
+
+.. code-block:: xml
+
+    <relatedItem displayLabel="Digital Collection" type="host">
+      <titleInfo>
+        <title>Children's Defense Fund</title>
+      </titleInfo>
+    </relatedItem>
+
+
```

Comment:
I'm assuming we'll also drop this as it is "synonymous with @displayLabel='Project'", but maybe adding one sentence would make this clearer (just saying this is the action that will be taken rather than referring to another use case).

Reply:
@mlhale7 thanks for this suggestion - I've added a note that hopefully helps clarify what we're doing here.